docs/ECLWatch/TheECLWatchMan.xml

```diff
@@ -63,20 +66,448 @@
   <chapter id="ECLWatch_Intro">
     <title>Introducing ECL Watch</title>
 
-    <sect1 id="ECLWatch_Introduction" role="nobrk">
-      <title>Introduction</title>
+    <para>ECL Watch is a service that runs on the Enterprise Services Platform
+    (ESP), a middleware component on the HPCC Platform.</para>
+
+    <para>ECL Watch provides an interface to the HPCC system and allows you to
+    view information and interrogate nodes to confirm all expected processes
+    are running. It is a plugin that is useful for Systems Administrators to
+    check processes, examine topology, and view logs. It is useful to ECL
+    Programmers to monitor the status of jobs and files, and other pertinent
+    information. This provides a simple view into the system and a means to
+    perform Workunit and data files maintenance. </para>
+
+    <para>The ECL Watch interface is a browser based set of pages where you
+    can access and interface with the HPCC System. To Run <emphasis
+    role="bold">ECL Watch</emphasis> using you browser, go to the node on your
+    system running <emphasis role="bold">ESP/ECL Watch</emphasis> on port
+    8010. For example, http://nnn.nnn.nnn.nnn:8010, where nnn.nnn.nnn.nnn is
+    your ESP/ECL Watch node's IP address. This will launch ECL Watch.</para>
+
+    <para>ECL Watch is organized by categories based on several system
+    functions. These functions or actions are accessible through the links
+    along the top of the main home page. However there are several ECL Watch
+    items with common functionality universally across pages of ECL
+    Watch.</para>
+
+    <sect1 id="Common_Elements">
+      <title>Common Elements</title>
+
+      <para>There are several common elements throughout the pages of ECL
+      Watch. These are very useful features that are available throughout the
+      various pages of ECL Watch. These common elements can work anywhere
+      within ECL Watch.</para>
```

Comment:
Replace 3 sentences with this one:
There are common elements that are useful anywhere within ECL Watch.